submissions/2025-05-21-OpenFF-CX3-CX4-singlepoints-v4.0/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenFF CX3-CX4 singlepoints v4.0
+
+## Description
+
+A dataset of single-point calculations generated to train bond and angle parameters for a test case of torsion drives for the t17 and t18 torsions in Sage 2.2.1. Conformers were generated following the process in https://github.com/lilyminium/refit-t17-t18-torsions/tree/main/01_generate-singlepoints .
+
+In short, for a particular molecular graph, a single conformer was generated. This molecule was assigned parameters from the Sage 2.2.1 force field. Additional torsion restraints of 1e5 kJ/mol were applied to restrain every torsion in the molecule. An MD simulation was then conducted at 500K and with 0.1 fs timestep. Frames were grouped by torsion similarity, and conformers were chosen from the biggest cluster.
+
+This process was repeated for every molecule in which a t17 or t18 torsion has been driven. Some additional small molecules were also included. Molecules for which the process only generated a single conformer were excluded from the dataset.
+
+This dataset is computed at the default OpenFF level of theory. The molecules here are all neutral and include the Br, C, Cl, F, H, I, N, O, S elements. They range from 30-307 Da (mean 162) and 2-21 heavy atoms. 
```

Comment:
It would be good to define the default LOT in case we ever change it.
```suggestion
This dataset is computed at the default OpenFF level of theory (B3LYP-D3BJ/DZVP). The molecules here are all neutral and include the Br, C, Cl, F, H, I, N, O, S elements. They range from 30-307 Da (mean 162) and 2-21 heavy atoms, all with a charge of zero. 
```